### PR TITLE
chore: better instructions for running benchwrapper

### PR DESCRIPTION
**Pull request by jadekler**

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ bencharmking purposes.
 ## Running
 
 ```
+npm install -g typescript
```
Comment (callmehiphop) [inline bin/README.md]: Why is a global install of TypeScript needed to run the benchmarks?

Comment (jadekler) [reply bin/README.md]: It's needed for `npm install` to work.

Comment (jkwlui) [reply bin/README.md]: It shouldn't be needed.. What error did you receive?

Comment (jadekler) [reply bin/README.md]: Tried to repro but couldn't. Closing.

**Pull request closed without merging**